Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/ListQuotasForIdentity/FakeMetricCalculatorFactory.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Backbone.Modules.Quotas.Domain;
+using Backbone.Modules.Quotas.Domain.Metrics;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Identities.ListQuotasForIdentity;
+
+public class FakeMetricCalculatorFactory : MetricCalculatorFactory
+{
+    private readonly int? _numberOfFiles;
+    private readonly int? _numberOfSentMessages;
+    private readonly int? _numberOfRelationships;
+    private readonly int? _numberOfRelationshipTemplates;
+    private readonly int? _numberOfTokens;
+    private readonly int? _amountOfUsedFileStorageSpace;
+
+    public FakeMetricCalculatorFactory(int? numberOfFiles = null, int? numberOfSentMessages = null, int? numberOfRelationships = null,
+        int? numberOfRelationshipTemplates = null, int? numberOfTokens = null, int? amountOfUsedFileStorageSpace = null)
+    {
+        _numberOfFiles = numberOfFiles;
+        _numberOfSentMessages = numberOfSentMessages;
+        _numberOfRelationships = numberOfRelationships;
+        _numberOfRelationshipTemplates = numberOfRelationshipTemplates;
+        _numberOfTokens = numberOfTokens;
+        _amountOfUsedFileStorageSpace = amountOfUsedFileStorageSpace;
+    }
+
+    protected override IMetricCalculator CreateNumberOfFilesMetricCalculator()
+    {
+        throw new NotImplementedException();
```

Comment:
You should implement all methods. It's weird that you can pass a parameter, and then it's never used.